echopype/consolidate/api.py

```diff
@@ -204,7 +204,7 @@ def sel_interp(var, time_dim_name):
     ).any()
     interp_msg = (
         "Interpolation may be negatively impacted, "
-        "consider handling these values before calling add_location."
+        "consider handling these values before calling ``add_location``."
```

Comment:
Do you know if the error/earning messages would be formatted if one uses ``x`` ? I know it would be rendered in docstrings, but not sure about the printout messages. I guess this is useful regardless because users would know the thing in the quote is a variable?